src/man/firejail.txt

```diff
@@ -2129,7 +2129,7 @@ $ firejail --read-only=~/test --read-write=~/test/a
 .TP
 \fB\-\-rlimit-as=number
 Set the maximum size of the process's virtual memory (address space) in bytes.
-you can use kilobyte(k),megabyte(m) and gigabyte(g) for size suffix. ( they works on base 1024 )
+use k(ilobyte), m(egabyte) or g(igabyte) for size suffix (base 1024).
```

Comment:
It's a capital U (not u) after completing a sentence:
Use k(ilobyte), m(egabyte) or g(igabyte) for size suffix (base 1024).
Please change this and we're set for merge.